app/models/views/applikation/result.rb

```diff
@@ -0,0 +1,28 @@
+module Views
+  module Applikation
+    class Result < Views::ApplicationResult
```

Comment:
Agreed, but maybe raise it as an issue and do it as a separate PR?  Looking at it, this will add an extra 9 files to this PR because of the model inheritance and subsequent tests